src/ansys/mechanical/core/embedding/app.py

```diff
@@ -116,13 +116,13 @@ def new(self):
 
     def close(self):
         """Close the application and remove the lock file."""
+        self.ExtAPI.Application.Close()
```

Comment:
@RobPasMue I think if it fails I'd rather the exception not get caught, since `close` doesn't know how the user wants to handle the error.  imo exceptions should only get caught in libraries if the library knows how to handle it.

Reply:
Okay, no try-except block in that case =)